control-plane-operator/controllers/hostedcontrolplane/manifests/csisnapshotcontroller.go

```diff
@@ -0,0 +1,35 @@
+package manifests
+
+import (
+	appsv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+	rbacv1 "k8s.io/api/rbac/v1"
+)
+
+func CSISnapshotControllerOperatorDeployment(ns string) *appsv1.Deployment {
+	dep := &appsv1.Deployment{}
+	dep.Name = "csi-snapshot-controller-operator"
+	dep.Namespace = ns
+	return dep
+}
+
+func CSISnapshotControllerOperatorRole(ns string) *rbacv1.Role {
+	role := &rbacv1.Role{}
+	role.Name = "csi-snapshot-controller-operator-role"
```

Comment:
nit: I'd put csi-snapshot-controller-operator into a constant a use it as the name for everything, -role is redundant with the type

Reply:
It already has this name in standalone OCP and I'd like to keep the names the same.